docs/documentation/release_notes/topics/22_0_0.adoc

```diff
@@ -1,3 +1,27 @@
+= Transition from Java EE to Jakarta EE
+
+We have migrated our codebase from Java EE (Enterprise Edition) to Jakarta EE, which brings various changes into Keycloak.
+We have upgraded all Jakarta EE specifications in order to support Jakarta EE 10.
+You can be affected by changing `javax.\*` to the `jakarta.*` namespace.
+
+See the migration guide for more details.
+
+= We upgraded to Quarkus 3!
+
+We have upgraded the Quarkus Java framework, which lives underneath Keycloak and brings many benefits.
+Quarkus 3 continues the tradition of propelling Java development by moving fast and providing a cutting-edge user experience with the latest technologies.
+It continues to improve overall performance and efficiency.
+
+It provides specific breaking changes, so check the migration guide for details.
```

Comment:
I'd maybe slightly rephrase.
```suggestion
Since Quarkus 3 is a major upgrade, some breaking changes were inevitable. See the migration guide for details.
```